### PR TITLE
TASK: Exchange yarn with make as build tool

### PR DESCRIPTION
**Pull request by mstruebing**
references #1629 

I would update the docs if we decide to merge it.
I would love to hear what you people think of this.

### CODE REVIEW

**Comment by dimaip** [comment]
Hey Max, thanks! I'm not opinionated about tooling at all, I'm fine with whatever works, but... why make? I've never seen it used in JS projects... What advatages does it bring over plain npm scripts?
In any case I think it might make sense to wrap its functionality via npm scripts anyways, no?

**Comment by mstruebing** [comment]
Rebased to remove `.nvmrc` and working fine :)

**Comment by dimaip** [comment]
Is this still WIP?
@johannessteu what do you think about this PR?

**Comment by mstruebing** [comment]
This is still WIP because I didn't update the docs according to this change. I want at least one approval to this change from someone who has an opinion to not decide alone on our build stack. :)

**Comment by johannessteu** [comment]
@dimaip @mstruebing i simply love it!

**Comment by dimaip** [comment]
@mstruebing so you get a +1 from @johannessteu and +0.(6) from me, merge it?

**Comment by mstruebing** [comment]
@dimaip first adjust the documentation ;)

I'm on it.

**Comment by mstruebing** [comment]
I'm done! :)

File: README.md [hunk 1]
```diff
@@ -111,6 +102,18 @@ Use `it.only(() => {})` and `describe.only(() => {})` if you want to run a speci
...
+Run `VERSION=<VERSION_YOU_WANT_TO_RELEASE> make release`
+e.g `VERSION=1.0.2 make release`.
+This checks if you set the environment variable, reinstall all node_modules,
+builds a production release runs `make lint`, `make test` and `make test-e2e`,
+bumps the version locally via lerna and publish the version as a new npm tag.
+After that you should carefully revisit your local changes, commit then and
+open a pull request on Github. When travis runs through merge it an tag a new
+release on Github.
+
+After that trigger jenkins with the new version.
```
Comment (dimaip) [inline README.md]: Could Jenkins do everything, including `make release`?

Comment (johannessteu) [reply README.md]: In general yeah. We need to give him access to master branch and should find some way to only commit stuff we expect to change

Comment (mstruebing) [reply README.md]: Master is generally protected, I don't know how to change that especially only for jenkins, we should not be able to push into master.

**Pull request merged**